mne_bids/copyfiles.py

```diff
@@ -223,6 +258,14 @@ def copyfile_brainvision(vhdr_src, vhdr_dest, verbose=False):
         The src path of the .vhdr file to be copied.
     vhdr_dest : str
         The destination path of the .vhdr file.
+    anonymize : bool
+        If False, leave dates in data files as they are. If True, replace all
+        date strings with 1924-01-01T00:00:00.
+    date : datetime.datetime | None
```

Comment:
this API is different from the other `anonymize` argument. Why is that?

Reply:
That is because the API for `anonymize` in `write_raw_bids` accepts a dict with a `daysback` int, and a `keep_his` bool.

Further up in the code, `daysback` is converted to a date, so I didn't want to duplicate that code once more in the `copyfile` call.

Lastly, I think it's good to have a way of anonymizing everything by just passing `True`. Else I would always have to inspect my script to remove the `daysback` int before sharing.

---

That being said, I *could* extend our `anonymize` dict to be also able to have an optional `date` key instead of just `daysback` and `keep_his`.

Then I could use that dict consistently and:
- `daysback` can be used for keeping intervals between subjects consistent but anonymous
- `date` as an alternative to `daysback` to just purge the information
- `keep_his` just as before: either delete all other subject information, or keep all (except date)


WDYT?

Reply:
> Lastly, I think it's good to have a way of anonymizing everything by just passing True. Else I would always have to inspect my script to remove the daysback int before sharing.

I don't understand this point. Does it mean that you remove the dates when `anonymize=True`?

I would just keep things consistent to avoid making the API a mess. No `date` parameter and a `daysback`parameter consistent with the other function. Consistent interfaces really pay off in the long run when users don't have to fight with errors because everything is the same.